docs/fiddles/accelerator/main.js

```diff
@@ -0,0 +1,28 @@
+// Adding a global shortcut to electron
+//
+// For more info, see:
+// https://electronjs.org/docs/api/screen
+
+const { app, globalShortcut, BrowserWindow } = require('electron')
+
+let mainWindow = null;
+
+app.on('ready', () => {
+  // Register a 'CommandOrControl+Y' shortcut listener.
+  globalShortcut.register('CommandOrControl+Y', () => {
+    // Do stuff when Y and either Command/Control is pressed.
+    console.log('shortcut...');
+  });
+  
+  mainWindow = new BrowserWindow({
+    width: 800,
+    height: 600,
+    webPreferences: {
+      nodeIntegration: true
+    }
```

Comment:
Since the goal is to provide running examples that people will probably copy-and-paste, IMO we shouldn't be enabling `nodeIntegration` in our fiddles unless we really have to.

This demo doesn't require any particular contents in the browser window. Could we load an index.html that doesn't require Node.js?

Reply:
✅ Turns out that was some silly copy paste I didn't notice. `nodeIntegration` isn't necessary at all for `index.html` to load, so I'll just remove that.